src/RedirectRequest.jl

```diff
@@ -71,4 +76,13 @@ function isdomainorsubdomain(sub, parent)
     return sub[length(sub)-length(parent)] == '.'
 end
 
+function verify_url(url::URI)
+    if isempty(url.scheme) || !(url.scheme in ("http", "https", "ws", "wss"))
```

Comment:
Any more schemes that can be valid here?

Reply:
ftp maybe?

Reply:
Doesn't look like it, unless I am missing something. I suppose this can be relaxed later if it turns out it breaks some usecase.